client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -194,6 +200,8 @@ class HandleEmailedLinkForm extends React.Component {
 
 const mapState = state => {
 	return {
+		redirectToFromQuery: getInitialQueryArguments( state ).redirect_to,
+		redirectToFromServer: getRedirectTo( state ),
```

Comment:
We could name this particular prop `redirectToSanitized` as well. What do you think?